relay-spans/src/v2_to_v1.rs

```diff
@@ -202,6 +209,58 @@ fn span_v2_link_to_span_v1_link(link: SpanV2Link) -> SpanLink {
     }
 }
 
+/// Generates a `sentry.op` attribute for V2 span, if possible.
+///
+/// This uses attributes of the span to figure out an appropriate operation name, inferring what the SDK might have sent. Reliably infers an op for well-known OTel span kinds like database operations. Does not infer an op for frontend and mobile spans sent by Sentry SDKs that don't have an OTel equivalent (e.g., resource loads). This relies on the SDK to send a `sentry.op` attribute, since we cannot infer an op.
+fn derive_op_for_v2_span(span: &SpanV2) -> String {
+    // NOTE: `op` is not a required field in the SDK, so the fallback is an empty string.
+    let op = String::from("default");
+
+    let attributes = if let Some(attributes) = span.attributes.value() {
+        attributes
+    } else {
+        return op;
+    };
+
+    let kind = span.kind.value().unwrap_or(&SpanV2Kind::Internal);
+
+    if attributes.get("http.request.method").is_some() || attributes.get("http.method").is_some() {
```

Comment:
`get(…).is_some()` is the same thing as `contains_key(…)`.
```suggestion
    if attributes.contains_key("http.request.method") || attributes.contains_key("http.method") {
```